package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Templates for articles on both iOS and Android next-gen apps",
   "main": "ArticleTemplates/",
   "scripts": {
-    "setup": "echo ArticleTemplates/assets/css/ >> .git/info/exclude && echo ArticleTemplates/assets/build/ >> .git/info/exclude",
+    "setup": "if [ ! -d .git/info ]; then  mkdir -p .git/info; fi; echo ArticleTemplates/assets/css/ >> .git/info/exclude && echo ArticleTemplates/assets/build/ >> .git/info/exclude",
```

Comment:
I added this line in to check if the info folder exists as I attempted a fresh install of the repo and yarn setup fell over as it didn't exist.